frontend/src/scenes/session-recordings/SessionRecordings.tsx

```diff
@@ -61,6 +62,15 @@ export function SessionsRecordings(): JSX.Element {
                     <>
                         {tab === ReplayTabs.Recent && !recordingsDisabled && (
                             <>
+                                <NotebookSelectButton
+                                    resource={{
+                                        attrs: {
+                                            filters: filters,
+                                        },
```

Comment:
Yeah it won't work from an "accurate finding" perspective. But it will show notebooks with a playlist in it which is better than nothing _for now_.

Just wanted a clear way to create a Notebook instead of a playlist.